evolver/history.py

```diff
@@ -18,7 +18,7 @@ def get(self, query):
 
 
 class HistoryServer(History):
-    def __init__(self, config = None):
+    def __init__(self, evolver=None, config = None):
```

Comment:
Ignore this, it's a bug but not directly related, just needed a fix so that the above test shows the update bug.